extensions/azurecore/src/azureResource/services/subscriptionService.ts

```diff
@@ -44,15 +43,13 @@ export class AzureResourceSubscriptionService implements IAzureResourceSubscript
 					gotSubscriptions = true;
 				}
 				else if (!account.isStale) {
-					const errorMsg = localize('azure.resource.tenantTokenError', "Failed to acquire Access Token for account '{0}' (tenant '{1}').", account.displayInfo.displayName, tenantId);
-					console.warn(errorMsg);
-					void vscode.window.showWarningMessage(errorMsg);
+					Logger.error(`Failed to acquire Access Token for account '${account.displayInfo.displayName}' (tenant '${tenantId}').`);
+					void vscode.window.showWarningMessage(`Failed to acquire Access Token for account '${account.displayInfo.displayName}' (tenant '${tenantId}').`);
```

Comment:
These showWarningMessage ones are still displayed to the user and should be localized